lib/graphql/resolvers.ts

```diff
@@ -57,6 +58,7 @@ const resolvers = {
     guardian: applicantGuardianResolver,
     medicalInformation: applicantMedicalInformationResolver,
     medicalHistory: applicantMedicalHistoryResolver,
+    recentPermit: applicantRecentPermitResolver,
```

Comment:
Sorry this is so far down in the review lol but can we rename this to `mostRecentPermit`? Makes it more clear